src/index.js

```diff
@@ -574,7 +677,12 @@ export default function (babel) {
     }
   }
 
+  // XXX: source mapping for inserted imports? not sure how to handle.
+  // guessing path here is just the top of the body which is maybe fine?
```

Comment:
ah, interesting... so many conflicting tools using the same info in different ways...

this is probably fine then for now. In the future maybe we can insert the auto-imported imports at the end of the `import` statements, which might help?